benchmarks/tests/numba_md_mod.py

```diff
@@ -93,11 +93,10 @@ def r8mat_uniform_ab ( r: 'double[:,:]', m: int, n: int, a: float, b: float, see
         seed = seed + i4_huge
 
     elif ( seed == 0 ):
-#        print ( '' ) # TODO error in Pyccel
+        print ( '' )
         print ( 'R8MAT_UNIFORM_AB - Fatal error!' )
         print ( '  Input SEED = 0!' )
-        # TODO must stop here
-#        sys.exit ( 'R8MAT_UNIFORM_AB - Fatal error!' )
+        sys.exit ( 'R8MAT_UNIFORM_AB - Fatal error!' )
```

Comment:
This change matches what pyccel can do, but doesn't really make sense in the benchmarks

Reply:
@yguclu Proposed solution:  f0d42c7